extra-dev/packages/coq-quickchick/coq-quickchick.8.10.dev/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+name: "coq-quickchick"
+maintainer: "llamp@seas.upenn.edu"
+synopsis: "Randomized Property-Based Testing Plugin for Coq"
+
+homepage: "https://github.com/QuickChick/QuickChick"
+dev-repo: "git+https://github.com/QuickChick/QuickChick.git"
+bug-reports: "https://github.com/QuickChick/QuickChick/issues"
+license: "MIT"
+
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "coq" {>= "8.10" < "8.11~"}
+  "coq-ext-lib"
+  "coq-mathcomp-ssreflect"
+  "ocamlbuild"
```

Comment:
Please use the `build` key whenever a dependency is only used during build time, e.g., for `ocamlbuild`. More information in the [OPAM documentation](https://opam.ocaml.org/doc/Packaging.html).

Reply:
`ocamlbuild` is a true dependency, called by our executable:
https://github.com/QuickChick/QuickChick/blob/8.10/quickChickTool/quickChickTool.ml#L365

Reply:
@liyishuai can you confirm that `menhir` is also a runtime dependency? I only see it used to produce `quickChickToolParser` during build time.

Reply:
I didn't see it in runtime either. @ejgallego could you confirm this?

Reply:
Maybe we can go with `"menhir" {build}` for now and fix later if problems show up. Changes look good.